content/docs/recipes/category-tables.md

```diff
@@ -61,7 +61,7 @@ Category Table Resources can be referenced by name in `categories` properties of
 }
 ```
 
-As with the [External Foreign Keys](/recipes/external-foreign-keys/) recipe, references to Category Table Resources in external packages can be made by providing the `categories` property an object with a `package` and `resource` property, both required `string`s. For example:
```

Comment:
Should we be a bit more specific about what is acceptable in the `resource` property, e.g., absolute path, URL, etc.?